### DISCUSSION TITLE
Make original event context available inside the

### DISCUSSION THREAD

**Discussion by thisYogesh** [vuejs/rfcs#288] [RFC Discussions]
# Links
- [RFC Proposal](https://github.com/thisYogesh/rfcs/blob/add-el-as-argument/active-rfcs/0000-add-%24el-argument.md)
- [PR](https://github.com/vuejs/vue/pull/11967)

**Comment by jods4** [comment]
> Currently in vue we don't have direct access to the element upon which event is attached

Isn't that `$event.currentTarget`?

**Reply by thisYogesh** [reply]
👀 I should have been checked that, Thanks @jods4!
